Dockerfile

```diff
@@ -1,64 +1,69 @@
-FROM ubuntu:18.04 as builder
+ARG ALPINE_VERSION=3.14
+ARG GO_VERSION=1.15-buster
+
+FROM golang:${GO_VERSION} AS builder
 
-# Install GCC
 RUN apt-get update && \
     apt-get install -y \
     gcc \
```

Comment:
as we are using `golang:1.15-buster` do we need to install gcc?

Reply:
That's how it was before, although we may want to remove it in our development branch Dockerfile since I think as well gcc comes pre-installed (to verify).